Doc/tutorial/errors.rst

```diff
@@ -119,9 +119,9 @@ may name multiple exceptions as a parenthesized tuple, for example::
    ... except (RuntimeError, TypeError, NameError):
    ...     pass
 
-A class in an :keyword:`except` clause is compatible with an exception if it is
-the same class or a base class thereof (but not the other way around --- an
-*except clause* listing a derived class is not compatible with a base class).
+A class in an :keyword:`except` clause is compatible with exceptions of the class
+itself and its derived classes. (but not the other way around --- an *except clause*
+listing a derived class is not compatible with exceptions of its base classes).
```

Comment:
```suggestion
A class in an :keyword:`except` clause matches exceptions which are instances of the
class itself or one of its derived classes. (but not the other way around --- an
*except clause* listing a derived class does not match instances of its base classes).
```

(if you scroll up it uses "matches" rather than "compatible with").